packages/cli/src/create/helpers/create-contract.ts

```diff
@@ -99,6 +103,23 @@ export async function createContractProject({
         `${chalk.green("Success!")} Created ${contractName} at ${contractPath}`,
       );
     }
+
+    if(!baseContract && createExtension) {
```

Comment:
wonder if ppl are gonna create project + extension in one go 🤔 guess it doesnt hurt to have it